hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceShipper.java

```diff
@@ -376,7 +376,7 @@ void clearWALEntryBatch() {
         totalToDecrement.accumulate(entrySizeExcludeBulkLoad);
       });
     });
-    if( LOG.isTraceEnabled()) {
+    if(LOG.isTraceEnabled()) {
```

Comment:
not opposed to fixing the whitespace here (though would generally avoid it since it's not close to your other changes). but if you're going to, i think we should fix the spacing between `if` and the opening parenthesis.